lib/linguist/languages.yml

```diff
@@ -44,6 +44,14 @@
   tm_scope: source.bsl
   ace_mode: text
   language_id: 0
+2D Array:
```

Comment:
This should actually be named `2-Dimensional Array`. BioWare's documentation consistently refers to this format as either _"2da"_ or _"2-dimensional array"_, but rarely _"2D Array"_.

Reply:
Changed it an reran the list grammars script.